ginkgo.Jenkinsfile

```diff
@@ -28,6 +28,9 @@ pipeline {
             steps {
                 sh "cd ${TESTDIR}; make tests-ginkgo"
             }
+            post {
+                sh "cd ${TESTDIR}; make clean-ginkgo-tests || true"
```

Comment:
I guess the previous examples didn't do this, but make has native change directory support, so you can alternatively do something like:

`sh make -C ${TESTDIR} clean-ginkgo-tests || true`